lib/Cleantalk/ApbctWP/FindSpam/UsersChecker.php

```diff
@@ -177,8 +188,8 @@ private static function removeUsersWithoutIPEmail(array $users)
             }
 
             $user_meta = self::getUserMeta($user->ID);
-
-            $user_ip    = ! empty($user_meta[0]['ip']) ? trim($user_meta[0]['ip']) : false;
+            $ip_of_user_meta = TT::getArrayValueAsString($user_meta, 'ip');
```

Comment:
A bug is here: `TT::getArrayValueAsString` here will find `$user_meta['ip']` but not `$user_meta[0]['ip']` needed.